### ISSUE TITLE
Circuit connections for pallets?

### ISSUE THREAD

**Issue by geekofalltrades** [Deadlock989/IndustrialRevolution#425]
**Describe the issue**
Neither wooden pallets nor tin pallets support circuit connections. Is this intentional?

I tried to use a pallet in this train refueling design:
```
0eNrNVNFuqzAM/Rc/w9RSSlse9iNXE0qJb2stJMgJvesq/v06MHUVQ2v3thdEYvucYx/DBfamw5bJBigvQLWzHso/F/B0sMrEu3BuEUqggA0kYFUTT3vFjMaQPaSNqo9kEfoEyGp8g3LZJ3cB/jmn0ab1EX24Kc36lwTQBgqEo5DhcK5s1+yRBfs7CQm0zkups5FX4NLtbvu0TuAsr3lRPK2FSRNjPebkUeiEILsSkPXIQe7uwG4GWJlcYGeqPR7ViRzH1Jq47ihUEtPX+r/EPlRfxnMiDp3cXOnHjFTZc5DmDjBy+KCiVYt4aFrFKkQqeB7CH3SMSldHZXUkDtKfTDJwh58ZQ3BIa8SGwbEIbsfB+ChrGR+M+tYE0nFm4tCBEe00supfesFxJ2QmjZVIrV+l0Xf8WInJpFfzyzA37WJq4pzYWVHZKEqWZV5E/pjdxdTu2y36BeY/7l/2Q5fWjw1o881nVszAFp+tNsqYFI1kM9Vp6wzeY4gWxDUcfinlzS8sAWnLD2XZdplvdtlmW+wWq0Xe9/8Blf6ncw==
```
It's intended to buffer one filled canister/barrel. When a train pulls into the station, the inserters are wired so that:
1. The inserter removing empty containers from the train is only enabled when at least one filled container is buffered.
2. The inserter putting filled containers in the train is only enabled while the inserter pulling empty containers off is moving at least one item.

Both inserters have hand size overridden to 1. The result is that the refueler can't remove an empty canister/barrel from the train and then have the train drive off while the canning/barreling machine is filling it, leaving that train short a canister/barrel. Since the only purpose of the crate in this design is to store a single filled canister temporarily and make that info available on the circuit network, a container with a single slot is ideal -- a wooden pallet. But the pallets don't support circuit connections.

**Factorio version, IR version**
* Factorio: 1.1.107.66234 (win64)
* IR3: 3.1.23

**Comment by Deadlock989** [comment]
Yes, it was intentional. Setting up the pixel offset for a circuit connector is a fiddly bit of partially-informed guesswork, and you also need a sensible place on the sprite for the connector to "attach to". Pallets are just too simple. If you require an inventory read via circuit network, you will need to use a chest container with the bar set to the desired inventory size (or rig your circuit so that inventory size doesn't matter).
